src/App.tsx

```diff
@@ -52,13 +52,13 @@ const App: React.FunctionComponent = () => {
         <Route path="/messages" element={<MessagesPage />} />
         <Route path="/bookmarks" element={<BookmarksPage />} />
         <Route path="/lists" element={<ListsPage />} />
-        <Route path="/profile" element={<ProfilePage tabKey="tweets" />} />
+        <Route path="/:id" element={<ProfilePage tabKey="tweets" />} />
         <Route
-          path="/profile/with_replies"
+          path="/:id/with_replies"
           element={<ProfilePage tabKey="tweets-with-replies" />}
         />
-        <Route path="/profile/media" element={<ProfilePage tabKey="media" />} />
-        <Route path="/profile/likes" element={<ProfilePage tabKey="likes" />} />
+        <Route path="/:id/media" element={<ProfilePage tabKey="media" />} />
+        <Route path="/:id/likes" element={<ProfilePage tabKey="likes" />} />
```

Comment:
we have problem
1) clear site data
2) not login user
3) click any user
4)choose like
we have route with blank id. And white page.
![image](https://user-images.githubusercontent.com/103836064/191898788-ef849142-6753-4e3b-99ca-a6dcf2c95328.png)
![image](https://user-images.githubusercontent.com/103836064/191898855-d2b7d9d9-ab89-4b24-b307-6675cd402986.png)
![image](https://user-images.githubusercontent.com/103836064/191898940-8ddb7f19-0368-4cf2-8bac-17eacc41455d.png)
![image](https://user-images.githubusercontent.com/103836064/191899051-1c68c68d-7c53-4ffd-ae0e-a997de7b325c.png)

Reply:
Question is why unauthorised user has access to tweets?

Reply:
Fixed:
![image](https://user-images.githubusercontent.com/103769012/191928592-c3d921e4-ce24-4b7d-9f8a-f208886ee201.png)